src/assets/css/main.scss

```
@@ -0,0 +1,85 @@
+
+.page-layout {
+    display: flex;
+    flex-direction: column;
+    min-height: 100%;
+}
+.main {
+    flex-grow: 1;
+    padding-bottom: 30px;
+}
+
+.u-mobile-only { display: none; }
+@include Mobile {
+    .u-mobile-only { display: block; }
+}
+
+.u-hidden {
+    display: none !important;
+}
+
+.u-visually-hidden {
+    position: absolute !important;
+    width: 1px !important;
+    height: 1px !important;
+    clip: rect(0 0 0 0);
+    padding:0 !important;
+    border:0 !important;
+    overflow: hidden;
```

I saw such [visually hidden class](https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html).
```suggestion
.visually-hidden {
  clip: rect(0 0 0 0); 
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap; 
  width: 1px;
}
```

---

Updated